components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled from 'styled-components'
+
+import '$web-components/app.global.scss'
```

Comment:
Q. Is this a right way to use variables?

Reply:
Ideally, I think we should add the import in our root container (maybe `brave/components/brave_new_tab_ui/containers/app.tsx`). Then you should be able to use the CSS variables anywhere in the new tab page :smile:

Reply:
Great! Works well :)